.gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+stages:
+  - build
+
+trigger_testbed:
+  stage: build
+  image:
+    name: registry.gitlab.sensirion.lokal/sensirion/docker/ubuntu:18.04-1.0.0
+  tags:
+    - docker
+  script:
+  - apt-get update && apt-get install -y curl
```

Comment:
use `apt-get -q` in both cases

`man apt-get`
> Quiet; produces output suitable for logging, omitting progress indicators

Reply:
Let's just update to the 1.1.0 Docker image which includes curl.